peachjam/js/components/Annotations/Annotation.vue

```diff
@@ -4,48 +4,47 @@
   >
     <div class="card">
       <div class="card-body">
-        <div class="d-flex justify-content-between align-items-baseline">
+        <div class="d-flex mb-1">
           <div>
             <small class="fw-bold text-muted">{{ annotation.user }}</small>
           </div>
-          <div class="dropdown text-end">
+          <div class="dropdown ms-auto">
             <a
-              id="dropdownMenuButton"
-              class="btn btn-link"
-              type="button"
+              class="dropdown-toggle"
+              href="#"
               data-bs-toggle="dropdown"
               aria-haspopup="true"
               aria-expanded="false"
-            >
-              <i class="bi bi-three-dots" />
-            </a>
-            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-              <a class="dropdown-item" role="button" @click="editAnnotation">{{ $t('Edit') }}</a>
-              <a class="dropdown-item" role="button" @click="deleteAnnotation">{{ $t('Delete') }}</a>
-            </div>
+            />
+            <ul class="dropdown-menu">
+              <li>
+                <a class="dropdown-item" role="button" @click="editAnnotation">{{ $t('Edit') }}</a>
+              </li>
+              <li>
+                <a class="dropdown-item" role="button" @click="deleteAnnotation">{{ $t('Delete') }}</a>
+              </li>
+            </ul>
           </div>
         </div>
-        <form @submit="saveAnnotation">
+        <form @submit.prevent="saveAnnotation">
           <textarea
             v-if="editing"
             v-model="annotation.text"
             class="form-control grow-text"
             required
           />
-          <p v-else>
```

Comment:
avoid using `p` outside of paragraphs of text -- it adds margin at the bottom which is almost never wanted.